content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -162,8 +162,6 @@ fail validation.
 				<p><strong>Allowed Values</strong></p>
 				<ul>
 					<li>Undefined/nil</li>
-					<li>Known list</li>
-					<li><code>0</code></li>
```

Comment:
I think the "Allowed values" list must remain the same, since the policy does still let you specify a known list, no? 

@tallclair do you think you could confirm, or am I wrong here?

Reply:
Thanks @shannonxtreme! From https://github.com/kubernetes/pod-security-admission/blob/master/policy/check_hostPorts.go#L66 it doesn't look like PSA allows a known list. I think that is only possible for 3rd party policy controllers outside of k8s.

Reply:
The built-in PodSecurityAdmission is one implementation of Pod Security Standards... it does not have a mechanism for allowlisting host ports

Other implementations do have a way to allowlist specific host ports.

I do think we need to keep "known list" in the allowed values, but I agree that's a subtle and sort of confusing distinction, so I'd be happy to find a way to clarify that.

Would adding something like this help?
```
<li>Known list (not supported by some implementations, like [Pod Security Admission](/docs/concepts/security/pod-security-admission/))</li>
```

Reply:
Done

Reply:
@liggitt is PSA the only built-in implementation of the Standards in the project? If so, we can be more specific and say "Not supported by the built-in Pod Security Admission controller"

Reply:
> @liggitt is PSA the only built-in implementation of the Standards in the project?

Yes

> If so, we can be more specific and say "Not supported by the built-in Pod Security Admission controller"

Works for me